apm-sniffer/apm-sdk-plugin/servicecomb-plugin/servicecomb-java-chassis-2.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/servicecomb/v2/ProducerOperationHandlerInterceptor.java

```diff
@@ -52,6 +52,13 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
         SpanLayer.asRPCFramework(span);
     }
 
+    private String getHeadValue(Invocation invocation, String key) {
+        if (invocation.getContext().containsKey("sw8")) {
```

Comment:
```suggestion
        if (invocation.getContext().containsKey(key)) {
```

Hard code `sw8` is wrong. We have more headers than this. Notice, your method is in the `while`.

Reply:
How can i know trace message in http header or servicecomb context? How many kind of key in skywalking like sw8

Reply:
What do you mean? Are you starting a new discussion?

Reply:
ok, i have given new push

Reply:
issues: https://github.com/apache/skywalking/issues/10422

Reply:
I don't mean the issue of GitHub. I mean what is the issue of codes. 
You just mentioned trace broken, but what you changed is very strange.